src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechRecognizer.java

```diff
@@ -559,7 +559,7 @@ public void unsetWakeWord() {
 
   @Override
   public ServiceConfig getConfig() {
-    AbstractSpeechRecognizerConfig config = (AbstractSpeechRecognizerConfig) initConfig(new AbstractSpeechRecognizerConfig());
+    AbstractSpeechRecognizerConfig config = new AbstractSpeechRecognizerConfig();

```

Comment:
abstract classes probably shouldn't have configs.  they can't be instantiated.

Reply:
I'm commenting it out for the moment ...
but I suspect what should happen is this should remain, and the derived speech recognizer should call this super.getConfig()